src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -180,25 +180,25 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
     }
   }
 
-  onCommentChanged = (event: SyntheticInputEvent<HTMLTextAreaElement>) => this.setState({ comment: event.target.value })
+  handleCommentChanged = (event: SyntheticInputEvent<HTMLTextAreaElement>) => this.setState({ comment: event.target.value })
```

Comment:
I'd vote for disabling this rule that forces us to name these callbacks `handleXyz`.

Reply:
I disabled the rule